vhost_user_fs/src/fs_cache_req_handler.rs

```diff
@@ -0,0 +1,23 @@
+use std::os::unix::io::RawFd;
+use vhost_rs::vhost_user::message::VhostUserFSSlaveMsg;
+use vhost_rs::vhost_user::{Result as VhostUserResult, SlaveFsCacheReq, VhostUserMasterReqHandler};
+
+/// Trait for virtio-fs cache requests operations.  This is mainly used to hide
+/// vhost-user details from virtio-fs's fuse part.
```

Comment:
The documentation says that the purpose of this trait is to hide vhost-user details from the fuse part but both methods of this trait take a `VhostUserFSSlaveMsg` and return `VhostUserResult` so it's not clear to me what details are actually being hidden.  Instead I would have expected the trait methods to take generic arguments (like the fd offset, cache offset, length, and flags) and then have the implementation of the method construct the `VhostUserFSSlaveMsg` out of them and then convert the `VhostUserResult` into an `io::Result`.

Reply:
Make sense, I'm going to update it per your suggestion.